second_response_to_reviewers.tex

```diff
@@ -0,0 +1,366 @@
+
+\begin{verbatim}
+We have a problem with reviewers declining to re review the paper, which
+forced us to invite first time reviewers for the revision.  The new reviewers
+have a number of problems with the manuscript.  In particular you need to work
+much harder to justify the relevance of your work to understanding iterated
+games.
+\end{verbatim}
+
+We open our second response to the reviews by highlighting that in the original
+review no comments were raised about the relevance of the work.
+\textbf{Specifically all previous reviewers highlighted that the work was
+noteworthy and worth publishing}.
+
+Regarding the relevance of the work, comments from each reviewer were:
+
+\begin{itemize}
+    \item 
+\begin{verbatim}
+The paper is well written, the results seem solid and overall very interesting.
+The main issues with this work, in my opinion, lie in the presentation:
+\end{verbatim}
+    \item 
+
+\begin{verbatim}
+The analysis in the paper is well designed and clearly explained. It does an
+excellent job of leveraging the work of many years and many different
+researchers to create new and significant results. And, the paper serves as a
+good model of how to make all code and data available, making the results easily
+reproducible and expandable to other researchers curious about the behaviour of
+additional strategies.
+\end{verbatim}
+    \item 
+
+\begin{verbatim}
+The reported experiments fill an important hole in the understanding of IPD
+evolution and are sorely needed. The authors also take pains to ensure the data
+are publicly available, which is highly appreciated.
+\end{verbatim}
+\end{itemize}
+
+Most requests were on the subject of
+presentation of the results as well as requests for specific pieces of further
+analysis. All of these were addressed.
+
+This latest set of comments from the reviewers are mainly of two types:
+
+\begin{enumerate}
+    \item Further minor requests for presentation modifications (specifically
+        the remaining reviewer from the first round only made requests of this
+        type).
+    \item Questions about the suitability/impact of the work.
+\end{enumerate}
+
+As we detail below we have addressed all requests of the first type. For the
+requests of the second type, from the two new reviewers who are perhaps not
+experts in the IPD, we have made some modifications to the manuscript but our
+main responses are to answer the questions posed by the reviewers.
+
+Specifically we describe how the Moran process is a well studied model of
+evolutionary processes and that this paper is a major contribution to the
+literature. It can be thought of as a tournament on Moran processes equivalent
+to the seminal work by Robert Axelrod in the 1980s.
+
+Robert Axelrod's work on the Prisoner's Dilemma tournaments (described in
+the literature review) which has been cited more than 34,000 times considered
+pair wise matches between strategies. Our work here provides a much more robust
```

Comment:
let's say "more computationally-intensive that accesses deeper questions of evolutionary stability" instead of "much more robust"

We can also mention that the Moran process for the IPD has been frequently studied from a theoretical perspective, and that our results extend that to strategies that are very difficult to study analytically.